core/common/src/MapLayerSettings.ts

```diff
@@ -190,13 +190,27 @@ export interface ImageMapLayerProps extends CommonMapLayerProps {
 
 }
 
+/** The target onto which to drape a model map layer.
+ * @beta
+ */
+export enum ModelMapLayerDrapeTarget {
+  /** Drape only onto the background map. */
```

Comment:
Do you mean "ellipsoid" or "globe"? The "background map" is imagery; the ellipsoid is the geometry onto which the background map and/model map layers are draped.

Reply:
We have changed this enum type from "BackgroundMap" to "Globe".